src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -305,7 +368,14 @@ ASR::Module_t* load_module(Allocator &al, SymbolTable *symtab,
     if (ltypes) return nullptr;
 
     if( compile_module ) {
-        mod1 = compile_module_till_asr(al, rl_path, infile, loc, err);
+        // mod1 = compile_module_till_asr(al, rl_path, infile, loc, err);
+        std::string cmd = "lpython -c --disable-main " + infile;
+        system(cmd.c_str());
```

Comment:
For now this is fine, but it is fragile (it will fail if `lpython` is not in path, or a different `lpython` gets picked up). We will later rework it to simply call the appropriate C++ functions directly to do the compilation.